ui/client/components/graph/node-modal/editors/expression/Editor.ts

```diff
@@ -69,18 +70,21 @@ export const editors: Record<editorTypes, EditorConfig> = {
     editor: () => RawEditor,
     hint: () => i18next.t("editors.raw.switchableToHint", "Switch to expression mode"),
     validators: (param, errors, fieldLabel) => simpleEditorValidators(param, errors, fieldLabel),
+    components: () => null,
   },
   [editorTypes.BOOL_PARAMETER_EDITOR]: {
     editor: () => BoolEditor,
     hint: (switchable) => switchable ? BoolEditor.switchableToHint() : BoolEditor.notSwitchableToHint(),
     switchableTo: (expressionObj) => BoolEditor.switchableTo(expressionObj),
     validators: (param, errors, fieldLabel) => simpleEditorValidators(param, errors, fieldLabel),
+    components: () => null,
   },
   [editorTypes.STRING_PARAMETER_EDITOR]: {
     editor: () => StringEditor,
     hint: (switchable) => switchable ? StringEditor.switchableToHint() : StringEditor.notSwitchableToHint(),
     switchableTo: (expressionObj) => StringEditor.switchableTo(expressionObj),
     validators: (param, errors, fieldLabel) => simpleEditorValidators(param, errors, fieldLabel),
+    components: () => null,
```

Comment:
why don't you pass default properties as separate object e.g.
```
const defaults = {
    validators: (param, errors, fieldLabel) => simpleEditorValidators(param, errors, fieldLabel),
    components: () => null,
}
...
[editorTypes.STRING_PARAMETER_EDITOR]: {
    ...defaults,
    editor: () => StringEditor,
    hint: (switchable) => switchable ? StringEditor.switchableToHint() : StringEditor.notSwitchableToHint(),
    switchableTo: (expressionObj) => StringEditor.switchableTo(expressionObj),
}
```

or at least reuse the same functions?

Reply:
Done